tools/Internalizer/Internalizer.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
```

Comment:
What properties did you need from common.props? If it's not required, we can leave this import out.